docs/guides/user-guide/importing-projects.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Importing projects
+description: This page outlines how you can import projects to StackBlitz.
```

Comment:
❓ This page doesn't have an intro so I needed to make it.

Reply:
Thank you!!